modules/class.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
```

Comment:
- [ ](Optional) I can see that you are repeating this code severally in this file,=. I would like to suggest that you put it a function and call it wherever you need it to be called. The same suggestion goes for this line of code below:

`localStorage.setItem('books', JSON.stringify(books))`